src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -2185,9 +2185,9 @@ bool isRefEscape
                         refKind = argRefKindsOpt[argIndex];
                     }
                     if (refKind == RefKind.None &&
-                        parameter?.RefKind == RefKind.In)
+                        parameter?.RefKind is RefKind.In or RefKind.RefReadOnlyParameter)
```

Comment:
Looking at the PR commit-by-commit, I expected to see a test change corresponding to this commit 1 change. Is it in another commit?

Reply:
Yeah, good question. I don't think the effect of this can be currently tested. The only difference the `refKind` assignment below has is whether it's `RefKind.None` or not (that's how it's used by the ref safety checks). So the inconsistency here would be just internal.